src/grandorgue/dialogs/GOMidiListDialog.cpp

```diff
@@ -69,8 +70,9 @@ GOMidiListDialog::GOMidiListDialog(
   buttons->Add(close);
   topSizer->Add(buttons, 0, wxALIGN_RIGHT | wxALL, 1);
 
-  for (unsigned i = 0; i < midi_elements->GetMidiConfiguratorCount(); i++) {
-    GOMidiConfigurator *obj = midi_elements->GetMidiConfigurator(i);
+  for (unsigned i = 0; i < midi_elements.size(); i++) {
```

Comment:
@oleg68 Couldn't we use the iterator like in other code parts as midi_elements is a vector?

Reply:
oooh, yes, didn't see

Reply:
But don't you have access to index on the vector one object ?

Reply:
I found you can use std::distance(vector.begin(), element).

It's nitpicking, just that I like iterator writing instead of indexes one. :-)

Reply:
But I saw distance() will do a loop, so no.